documentation/endpoints/v3/posts.md

```diff
@@ -140,8 +140,8 @@ POST /api/v3/posts
     The Drupal/Contentful ID of the campaign that the user's post is associated with.
   - **campaign_run_id**: (int) optional.
     The drupal campaign run node id of the campaign that the user's post is associated with.
-  - **quantity**: (int).
-    The number of reportback nouns verbed. 
+  - **quantity**: (int|nullable).
```

Comment:
should we note it is optional like we did for `campaign_run_id`?

Reply:
love this idea! added :)